tests/cypress/e2e/actions_tasks2/case_111_settings_text_size_position_label_content.js

```diff
@@ -10,6 +10,7 @@ import {
     labelName,
     attrName,
     textDefaultValue,
+    skeleton,
```

Comment:
I already wrote the suggestion to create in headless by default, but it is optional. 
The minimum changes would be to create a task in headless mode within this test